can-type.js

```diff
@@ -219,6 +219,34 @@ var Any = canReflect.assignSymbols({}, {
 	"can.isMember": function() { return true; }
 });
 
+function all(typeFn, Type) {
+	var typeObject = typeFn(Type);
```

Comment:
Is there a reason the `Type` needs to be passed through the converter also? (there probably is, just want to be sure)

Reply:
It's possible that this is not needed, let me check...

Reply:
Oh, we do want the isMemberSymbol. It might be better to make this return a typeObject and not a constructor, that way people don't attempt to use `instanceof` or something.

Reply:
After thinking about it, I left this alone because we want to inherit the `strictTypeOf` symbol and anything else that is on the typeObject. Because this is a constructor function it will inherit those things.  Type also might be a `typeObject` itself.